static/js/utils/dev_flag_utils.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** URL flags for gating features under development */
+
+import queryString from "query-string";
+
+export const DEV_FLAGS = {
+  // Temporary flag to gate new chart icons in the footer
+  // TODO (juliawu): Remove this flag once all chart action icon changes are in.
+  USE_CHART_ACTION_ICONS_FLAG: "chartIcons",
```

Comment:
Maybe prefix flags with something like "experiment" so its clear in the url params that it's for an experiment?